iron-request.html

```diff
@@ -443,11 +443,10 @@
     _wwwFormUrlEncodePiece: function(str) {
       // Spec says to normalize newlines to \r\n and replace %20 spaces with +.
       // jQuery does this as well, so this is likely to be widely compatible.
-      if (str === null) {
-        return '';
-      }
-      return encodeURIComponent(str.toString().replace(/\r?\n/g, '\r\n'))
-          .replace(/%20/g, '+');
+      var castedStr = new String(str);
```

Comment:
I _think_ the only values to be wary of here are `null` and `undefined`. If that is true, a loose equality check should be sufficient. For example:

```
if (str == null) {
  return new String(str).toString();
}
```

Although it is less obvious, a more concise and technically similar return value could be `str + ''`, which will coerce `null` or `undefined` to their string representations.